README.md

```diff
@@ -98,16 +86,7 @@ Install this project with:
   cd my-project
   gem install
 ```
--
 
-### Usage
-
-To run the project, execute the following command:
-
-```sh
-  rails server
```

Comment:
- Kindly include the usage section and replace `rails server` with `ruby main.rb`